src/topology/subset_properties.lean

```diff
@@ -1131,6 +1133,12 @@ instance locally_compact_space.pi [∀ i, compact_space (π i)] : locally_compac
     { rw if_neg h, exact compact_space.is_compact_univ, } },
 end⟩
 
+instance locally_compact_space.function_finite [finite ι] [locally_compact_space β] :
+  locally_compact_space (ι → β) := locally_compact_space.pi_finite
+
+instance locally_compact_space.function [locally_compact_space β] [compact_space β] :
+  locally_compact_space (ι → β) := locally_compact_space.pi
```

Comment:
Should we call these `function.locally_compact_space_of_finite`, `function.locally_compact_space`? We should then also rename the three existing instances ([locally_compact_space.pi_finite](https://leanprover-community.github.io/mathlib_docs/topology/subset_properties.html#locally_compact_space.pi_finite) etc.) similarly.

Reply:
Indeed, the existing instances are misnamed.